database/tables/workspace_logs.pg

```diff
@@ -3,8 +3,8 @@ columns
     id: bigint not null default nextval('workspace_logs_id_seq'::regclass)
     level: enum_log_level
     message: text
```

Comment:
Where is the state logged?

Reply:
State changes were always logged in `workspace_logs.message`. We created that column before we actually had a "message" which is now a bit confusing.